### ISSUE TITLE
Stream with a Pi Camera on Decaffeine

### ISSUE THREAD

**Issue by birilo** [melnijir/Dicaffeine#35]
Hi, I search to stream with my pi cam through Decaffeine.
The pi cam is not recognyze on Decaffeine whereas it's recognyze with libcamera. 
Any idea for help me?

Thanks to you

**Comment by bcnine** [comment]
Yes this is a pretty obvious use case, streaming from a camera attached to the Rpi dedicated port. Probably should mention that this isn't supported directly in the README, would save people some time.

**Comment by Mavyre** [comment]
Anybody has made that work, by any chance?

**Comment by johnedstone** [comment]
I've gotten dicaffeine to work with the PI Camera on bullseye by enabling (raspi-config) Interface Options --> I1 Legacy Camera
```
###Working in the past on this:
$ cat /etc/os-release 
PRETTY_NAME="Debian GNU/Linux 11 (bullseye)"
NAME="Debian GNU/Linux"
VERSION_ID="11"
VERSION="11 (bullseye)"
VERSION_CODENAME=bullseye
ID=debian

$ apt  list --installed dicaff*
Listing... Done
dicaffeine/unknown,now 0.6.3 arm64 [installed]
```
